src/bundle/Resources/public/css/modules/trash.css

```diff
@@ -0,0 +1,33 @@
+.ez-trash-server-side-content {
+    display: flex;
+}
+
+.ez-trash-view {
+    display: flex;
+    flex: 1 0;
+}
+
+.ez-trash-view .ez-trash-view-main {
+    padding: 1em 2em 1em 2em;
+    flex: 1 0;
+}
+
+.ez-trash-view .ez-action-bar {
+    flex: 0 0 var(--ez-toolbar-width);
+    background: var(--ez-color-action-bar);
```

Comment:
there are the exact same styles in content-view.css. While it's ok for the `flex` one since it kind of depends on `.ez-trash-view` or `ez-content-view`, for the background and color, this should be avoided, so it's time to create an `action-bar.css` that defines those